stub.bash

```diff
@@ -1,6 +1,9 @@
 BATS_MOCK_TMPDIR="${BATS_TMPDIR}"
 BATS_MOCK_BINDIR="${BATS_MOCK_TMPDIR}/bin"
 
+MKDIR=$(which mkdir)
```

Comment:
Possibly prefix commands with `BATS_MOCK` to avoid name clashes with user variables.